brainscore_vision/models/mobilenet_v2_1-4_224_pytorch/requirements.txt

```diff
@@ -0,0 +1,3 @@
+torchvision
+torch
```

Comment:
Are we leaving in torch in requirements.txt? Not sure what was decided @deirdre-k

Reply:
For now yeah. Once tests are migrated and the conda env approach in ironed out I think we should move it to a project-level dependency, but until then we want to leave it. Thanks for remembering!

Reply:
Sounds good - @pellegreene feel free to ignore my other PR comments regarding this then